src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,14 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
+    <h2 className="TodoInfo__title">{todo.title}</h2>
```

Comment:
You can perform such destructuring in all components
```suggestion
    <h2 className="TodoInfo__title">{title}</h2>
```